api/apps/api/src/modules/scenarios/run.service.spec.ts

```diff
@@ -0,0 +1,252 @@
+import { EventEmitter } from 'events';
+import { PromiseType } from 'utility-types';
+import { right, left } from 'fp-ts/Either';
+import waitForExpect from 'wait-for-expect';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { Test } from '@nestjs/testing';
+import { API_EVENT_KINDS } from '@marxan/api-events';
+import {
+  notFound,
+  runEventsToken,
+  runQueueToken,
+  RunService,
+} from '@marxan-api/modules/scenarios/run.service';
+import { ApiEventsService } from '@marxan-api/modules/api-events/api-events.service';
+import { Scenario } from './scenario.api.entity';
+
+let fixtures: PromiseType<ReturnType<typeof getFixtures>>;
+let runService: RunService;
+
+beforeEach(async () => {
+  fixtures = await getFixtures();
+  runService = fixtures.getRunService();
+});
+
+test(`scheduling job`, async () => {
+  fixtures.schedulingJob();
```

Comment:
digging proves it prepares `fakeQueue` and `fakeApiEvents` mocks but the name does not imply it at all.

Shouldn't it be a part of every test as a part of the setup? 🤔

Reply:
I didn't want to bloat the code of the test here, I can change the name to `prepareMockForJobScheduling`

Reply:
7e05c206